### PR TITLE
fix: fix average precision evaluator

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

File: evaluators/rank/AveragePrecision/__init__.py [hunk 1]
```diff
@@ -7,30 +7,30 @@ class AveragePrecisionEvaluator(BaseRankingEvaluator):
...
     """
+
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     def evaluate(self, actual: Sequence[Any], desired: Sequence[Any], *args, **kwargs) -> float:
         """"
         :param actual: the matched document identifiers from the request as matched by jina indexers and rankers
-        :param desired: the expected documents matches ids sorted as they are expected
+        :param desired: the expected documents matches ids sorted as they are expected. All documents in this list are
```
Comment (JoanFM) [inline evaluators/rank/AveragePrecision/__init__.py:17]: this needs to be changed, order does not matter

File: evaluators/rank/AveragePrecision/manifest.yml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 manifest_version: 1
-name: AveragePrecision
+name: AveragePrecisionEvaluator
 kind: evaluator
 description: |
-  Calculate Average Precision.
+  Computes the AveragePrecision ranking metric for the resulting ranking of a Search System.
 author: Kilian Pfeiffer (kilsen512@gmail.com)
 url: github.com/kilsenp
```
Comment (bwanglzu) [inline evaluators/rank/AveragePrecision/manifest.yml:7]: what is this url and author?

File: evaluators/rank/AveragePrecision/README.md [hunk 3]
```diff
@@ -1,3 +1,74 @@
 # AveragePrecisionEvaluator
-Calculate Average Precision between two rankings as defined for [information retrieval](https://en.wikipedia.org/wiki/Evaluation_measures_(information_retrieval)#Average_precision).
 
+`AveragePrecisionEvaluator` computes the [Average Precision](https://en.wikipedia.org/wiki/Evaluation_measures_(information_retrieval)#Average_precision) for the resulting ranking of a 
+Search System.
```
Comment (bwanglzu) [inline evaluators/rank/AveragePrecision/README.md:4]: resulting ranking -> ranking result of a search system should be better?

**Review by bwanglzu — CHANGES_REQUESTED** [review]
Good one, only minor things

File: evaluators/rank/AveragePrecision/manifest.yml (hunk 2 above)
Comment (JoanFM) [reply evaluators/rank/AveragePrecision/manifest.yml:7]: the original author of this `executor`

File: evaluators/rank/AveragePrecision/README.md (hunk 3 above)
Comment (JoanFM) [reply evaluators/rank/AveragePrecision/README.md:4]: can u do the suggestion here in place so that I can commit directly?

**Review by bwanglzu — APPROVED** [review]
LGTM!

**Pull request merged**